Packs/CommonScripts/Scripts/diplayMappedFields/diplayMappedFields.py

```diff
@@ -0,0 +1,90 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+incident = demisto.incident()
+fields = incident.get('CustomFields', {})
+
+data = fields
```

Comment:
Please move it to the main function as well.
```suggestion
incident = demisto.incident()
data = incident.get('CustomFields', {})
```